README.md

```diff
@@ -0,0 +1,19 @@
+# Transmute.industries pubs
+
+oneliner to authorize Transmute team  to a server
+```
+curl -Ls git.io/tstaddus|bash
```

Comment:
Not as familiar with bash, but when I run this command I get `bash: line 1: 404:: command not found`.

Reply:
this is a redirect to:
```
Location: https://raw.githubusercontent.com/transmute-industries/pubs/master/addus
```
and will not exist till this pull request goes through